setup.py

```diff
@@ -3,66 +3,46 @@
 import os
 import sys
 
-import jwt
-
-from setuptools import setup
-from setuptools.command.test import test
-
-
-class PyTest(test):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
-
-    def initialize_options(self):
-        test.initialize_options(self)
-        self.pytest_args = []
-
-    def finalize_options(self):
-        test.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
-
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        errno = pytest.main(['.'] + self.pytest_args)
-        sys.exit(errno)
+from setuptools import find_packages, setup
 
+__version__ = None
+with open('jwt/__init__.py') as fp:
+    _locals = {}
+    for line in fp:
+        if line.startswith('__version__'):
+            exec(line, None, _locals)
+            __version__ = _locals['__version__']
+            break
```

Comment:
The import happens before any of the project dependencies are installed and it will fail in case they are missing. A better variant og the above is to have a _version.py file in the project with the version and 'exec' it.